oidc-conformance-tests/configure_is.py

```diff
@@ -374,6 +374,14 @@ def generate_config_for_plan(service_provider1_config, service_provider2_config,
     },
     constants.BASE_URL + "/api/server/v1/claim-dialects/aHR0cDovL3dzbzIub3JnL29pZGMvY2xhaW0/claims/d2Vic2l0ZQ")
 
+#change middle_name from middleName to formattedName
+change_local_claim_mapping(
+    {
+        "claimURI": "middle_name",
+        "mappedLocalClaimURI": "http://wso2.org/claims/formattedName"
+    },
+    constants.BASE_URL + "/api/server/v1/claim-dialects/aHR0cDovL3dzbzIub3JnL29pZGMvY2xhaW0/claims/bWlkZGxlX25hbWU")
+
 edit_scope("openid", {
```

Comment:
Is this step needed in the latest setup?

Reply:
This mapping is to prevent the warning that is caused because of missing middle_name claim (Related to https://github.com/wso2/product-is/issues/10638). 
These changes will configure the SP with this claim and fix this WARNING.